tachyon/math/base/batch_inverse_benchmark.cc

```diff
@@ -37,7 +37,7 @@ void BM_InverseParallelFor(benchmark::State& state) {
       state.range(0), [](size_t i) { return F::FromBigInt(BigInt(i + 1)); });
   for (auto _ : state) {
     OPENMP_PARALLEL_FOR(size_t i = 0; i < fields.size(); ++i) {
-      fields[i].InverseInPlace();
+      static_cast<void>(fields[i].InverseInPlace());
```

Comment:
This is an old pattern. include `<tuple>` and change like below. See also See https://stackoverflow.com/a/77195811

```suggestion
    std::ignore = fields[i].InverseInPlace();
```